### ISSUE TITLE
Some contacts apparently can't be fetched/followed

### ISSUE THREAD

**Issue by cpbs42** [friendica/friendica#12841]
I have searched open and closed issues and haven't found anything that looks like this.  I may not be picking search terms very well, of course.

Bug Description: Some contacts stay "blank" - meaning when I view the contact there is very little information (see screenshot below) - it doesn't look like anything has been populated into the contact from the origin site; when I try to "follow" the contact I get a blank page.

Steps to Reproduce

1. Use the box at the top of the page to search for a contact such as (a few examples which trigger this):

https://mastodon.social/users/jwz
https://botsin.space/users/carmensandiego
https://goblin.technology/users/tobi

2. Sometimes (I think the first time this friendica node has seen the contact) get a blank page as a result of the search, or other times I get a contact page with little info on it, only the "network" and "profile url" are filled in.

3. Do any of the following and get a blank page:
 - follow
 - point to "actions" then click "follow"
 - point to "actions" then click "re-fetch contact data"

Actual Result: as described above.  Most contacts populate fine and can be followed with no problem; contacts exhibiting the above symptoms are rare so my guess is something in the contact info returned by the remote server, but I'm not sure where to start on debugging - I've not been able to find anything useful in the logs.

Expected Result: non-blank pages, more contact info, following the contact.

Screenshots:

![friendica-contact-bug](https://user-images.githubusercontent.com/47782255/221325101-f9f6e6be-abb8-4f60-ba17-1ebb0c490570.png)

Platform Info:

Friendica Version: 2023.01
Friendica Source: (not sure what this refers to)
PHP version: 7.4.33 (running in a minimal chroot environment)
SQL version: 10.5.15-MariaDB-0+deb11u1 Debian 11
Contents of chroot: busybox and its aliases, esmtp, minimal /etc/passwd, /etc/resolv.conf, /dev/{null,zero,full,urandom,tty,fd,stdin,stdout,stderr} /usr/share/zoneinfo /etc/ssl /usr/share/ca-certificates /usr/lib/ssl PHP 7.4
Web server: PHP-FPM with chroot option turned on, behind Nginx 1.18.0-6.1, behind HAProxy 2.2.9-2+deb11u3

**Comment by MrPetovan** [comment]
White pages are likely to produce error messages in the PHP-FPM log. Additionally, you can enable Friendica logging that catches several PHP-related errors that would otherwise be silenced when the debug log is disabled.

**Comment by cpbs42** [comment]
Thank you for the help!  There's nothing useful in the PHP-FPM logs, but thanks to your mention of enabling logging I have found the debug log options in the admin pages.  Using that I got some error messages which pointed me towards some ImageMagick-related files (in /etc/ImageMagick-6, /usr/lib/x86_64-linux-gnu/ImageMagick-6.9.11 and /usr/share/ImageMagick-6) which were missing from my chroot environment. Adding those to my chroot-making script fixed my problem! Thanks. :)
